assets/src/components/AvatarModal.js

```diff
@@ -94,14 +95,23 @@ function AvatarModal (props) {
   )
 
   useEffect(() => {
-    const helpUrlContext = url.includes('grades')
-      ? '/grade-distribution'
-      : url.includes('assignment')
-        ? '/assignment-planning'
-        : url.includes('resources')
-          ? '/resources-accessed'
-          : ''
-    setHelpURL(`${helpURL}${helpUrlContext}`)
+    const getContextHelpUrl = (url) => {
+      let target
+      if (url.includes('grades')) {
+        target = viewHelpURLs.gd
+      } else if (url.includes('assignmentsv1')) {
+        target = viewHelpURLs.apv1
+      } else if (url.includes('assignments')) {
+        target = viewHelpURLs.ap
+      } else if (url.includes('resources')) {
+        target = viewHelpURLs.ra
+      } else {
+        target = viewHelpURLs.home
+      }
+      return target || viewHelpURLs.home
```

Comment:
All of the help urls can be independently configured so it's possible that any one of them is undefined.
Currently they all have (U of M) defaults except the new assignment planning.

So out of the box viewHelpURLs.ap is undefined and in that case it will use the 'default' help home page.

The default help home page could also be overridden to be undefined, which would just not work well.  But someone would have to do that purposefully.. we're planning to address removing u of m defaults and handling lack of configured help in the future.